Jenkinsfile

```diff
@@ -331,22 +332,53 @@ pipeline {
               }
             }
           }
-          stage('Release to NuGet') {
-            options { skipDefaultCheckout() }
+          stage('Release') {
```

Comment:
Created a top-level release stage which contains:
- a nested `Notify` stage to send an email with the link of the input approval. (NEW)
- a nested `Release to NuGet` stage as used to be.
- a new nested `opbeans` stage to bump and release the opbeans-dotnet project. (NEW)